web/client/translations/data.de-DE

```diff
@@ -54,6 +54,41 @@
             "tooltipDeactivate": "Beenden Sie den 3D-Modus",
             "tooltipActivate": "Geben Sie den 3D-Modus ein"
         },
+        "cookie":{
+            "info": "This website uses cookies to improve your experience. We'll assume you're ok with this, but you can opt-out if you wish.",
+            "moreDetailsButton": "More Details",
+            "leave": "Leave",
+            "accept": "Accept",
+            "moreDetails": {
```

Comment:
I saw i cannot put many tags inside a single translation. i.e. the whole page.  so i decided to put all the pieces. your suggestion require  page per language we will add, isn't it?